src/Uno.Extensions.Navigation.Generators/ForceBindingsUpdateGenerator.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Diagnostics;
+using Microsoft.CodeAnalysis;
+using Uno.Extensions.Generators;
+
+namespace Uno.Extensions.Navigation.Generators;
+
+/// <summary>
+/// A generator that generates bindable VM for the reactive framework
+/// </summary>
+[Generator]
+public partial class ForceBindingsUpdateGenerator : ISourceGenerator
```

Comment:
If it's not much work, I think it makes more sense to implement this as an IIncrementalGenerator. It would be more difficult though, and in this case, caching should be properly tested.

Reply:
can look at this in future if needed